frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/option_action.dart

```diff
@@ -159,9 +273,61 @@ class OptionActionWrapper extends ActionCell {
       case OptionAction.color:
         description = LocaleKeys.document_plugins_optionAction_color.tr();
         break;
+      case OptionAction.align:
+        description = LocaleKeys.document_plugins_optionAction_align.tr();
+        break;
       case OptionAction.divider:
         throw UnimplementedError();
     }
     return description;
   }
 }
+
+class OptionAlignWrapper extends ActionCell {
+  OptionAlignWrapper(this.inner);
+
+  final OptionAlignType inner;
+
+  @override
+  Widget? leftIcon(Color iconColor) {
+    var name = '';
+    // TODO: add icons.
+    switch (inner) {
+      case OptionAlignType.left:
+        name = 'editor/align/left';
+        break;
+      case OptionAlignType.center:
+        name = 'editor/align/center';
+        break;
+      case OptionAlignType.right:
+        name = 'editor/align/right';
+        break;
+      default:
+        throw UnimplementedError();
+    }
+    if (name.isEmpty) {
+      return null;
+    }
+    return FlowySvg(name: name);
```

Comment:
```suggestion
    late final String name;
    // TODO: add icons.
    switch (inner) {
      case OptionAlignType.left:
        name = 'editor/align/left';
        break;
      case OptionAlignType.center:
        name = 'editor/align/center';
        break;
      case OptionAlignType.right:
        name = 'editor/align/right';
        break;
    }
    return FlowySvg(name: name);
```
- All the cases should be covered here so there's no need for default IIUC. 
- Throw StateError or ArgumentException instead of UnimplementedError
- If name is a `late final String` then there's no need to add the `.isEmpty` condition at the end. It was impossible anyway since all the cases were covered in the switch statement.

Reply:
If every OptionAlignType have the asset (enhanced enum), then this whole class can be removed.

Reply:
I removed the switch case. Like @Xazin said, every align type have their own asset name. So I just added a assetName getter to align type.

```dart
  @override
  Widget? leftIcon(Color iconColor) {
    return FlowySvg(name: inner.assetName);
  }
```